iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/helpers/ExportImportDevicesParser.java

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package tests.integration.com.microsoft.azure.sdk.iot.helpers;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonSyntaxException;
+import com.google.gson.annotations.SerializedName;
+import com.microsoft.azure.sdk.iot.deps.serializer.ExportImportDeviceParser;
+
+import java.util.ArrayList;
+
+public class ExportImportDevicesParser {
+    private static final String DEVICES = "devices";
+    @SerializedName(DEVICES)
+    Iterable<ExportImportDeviceParser> exportImportDevices;
+    private transient static Gson gson = new Gson();
+
+    /**
+     * Converts this into json and returns it
+     * @return the json representation of this
+     */
+    public String toJson()
+    {
+        String json = gson.toJson(exportImportDevices);
+        return json;
+    }
+
+    /**
+     * Empty constructor: Used only to keep GSON happy.
+     */
+    public ExportImportDevicesParser()
+    {
+        exportImportDevices = new ArrayList<>();
+    }
+
+    /**
+     * Constructor for an ExportImportDevicesParser that is built using the provided json
+     * @param json the json string to build the ExportImportDevicesParser out of
```

Comment:
minor: "the json string used to build the ExportImportDevicesParser" might sound better.